src/main/java/com/n26/statistics/exceptions/TransactionInvalidException.java

```diff
@@ -0,0 +1,10 @@
+package com.n26.statistics.exceptions;
+
+public class TransactionInvalidException extends RuntimeException {
```

Comment:
we can even use @ResponseStatus(HttpStatus.NO_CONTENT)
and avoid global exception handler